ga4gh/cli.py

```diff
@@ -63,6 +64,51 @@ def server_main():
         app.backend = backend
         app.run(host="0.0.0.0", port=args.port, debug=True)
 
+
+def server_benchmark():
```

Comment:
This should probably be `server_benchmark_main()`, just to keep the convention that the main entry point for any program will end with `main`. (We should camelCase this convention at some point though)